common/components/ui/Spinner.tsx

```diff
@@ -11,7 +11,7 @@ interface SpinnerProps {
 const Spinner = ({ size = 'x1', light = false }: SpinnerProps) => {
   const color = light ? 'Spinner-light' : 'Spinner-dark';
   return (
-    <svg className={`Spinner Spinner-${size} ${color}`} viewBox="0 0 50 50">
+    <svg className={`Spinner Spinner-${size} ${color}`} viewBox="0 0 50 50" aria-busy="true">
```

Comment:
I like the idea of this, but this is actually supposed to mark the container as busy, not the loader. Perhaps this should mark the parent element as busy, and unmark it when this component unmounts?

Reply:
That might be a little bit difficult to properly find the correct parent element every time the spinner is mounted and unmounted. I think it would make more sense to just add the `aria-busy` attribute manually to all the parent elements that use the spinner.

Just like spinners are conditionally rendered with `{isLoading && <Spinner />}`, `aria-busy={isLoading}` can be added to the parent element.

Reply:
Yeah I think that'd have to be how it's done. I'm not sure we can easily enforce that, but I'm also not sure how good the benefits are for a screen reader.